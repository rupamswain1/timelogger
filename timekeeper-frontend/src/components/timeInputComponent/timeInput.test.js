import {mount} from 'enzyme';
import React from 'react';
import TimeInput from './timeInput.component';
import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

Enzyme.configure({ adapter: new Adapter() });

it('expect to render the Time input component which is not disabled',()=>{
    expect(mount(<TimeInput id='0'/>)).toMatchSnapshot();
})
it('expect to render the Time input component which is disabled',()=>{
    expect(mount(<TimeInput id='1'/>)).toMatchSnapshot();
})