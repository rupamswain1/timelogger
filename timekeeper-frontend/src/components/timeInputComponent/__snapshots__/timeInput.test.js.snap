// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render the Time input component which is disabled 1`] = `ReactWrapper {}`;

exports[`expect to render the Time input component which is not disabled 1`] = `ReactWrapper {}`;
